vtex-sites/base.store
# Adjust Incentives to render CMS data

## What's the purpose of this pull request?

This PR aims to keep the `Incentives` component synced after changes made in the forked project [storeframework](https://github.com/vtex-sites/storeframework.store/pull/47).

## How does it work?

This PR used cherry-pick to copy code updates from [storeframework](https://github.com/vtex-sites/storeframework.store/pull/47), and mocked the Incentives in `incentivesMock.js `file.

## How to test it?

Nothing special to be tested, just run the project and see the `IncentivesHeader` and `IncentivesFooter` working properly.

## References

- Storeframework [PR](https://github.com/vtex-sites/storeframework.store/pull/47) with final integration with CMS
- Related [Jira Task](https://vtex-dev.atlassian.net/browse/FSSS-261)
- Git cherry-pick [documentation](https://git-scm.com/docs/git-cherry-pick)

## Checklist
